app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -72,25 +70,26 @@
 
 <section class="col col-space-m col-720">
 <h2>Check your qualifications</h2>
-<p>To train to teach in primary and secondary schools in England, you’ll need:</p>
+<p>To train to teach in secondary schools in England, you’ll need:</p>
 <ul>
-<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
+<li>GCSEs at grade 4 (C) or above in English and maths</li>
 <li>a bachelor’s degree in any subject</li>
 </ul>
 <p>Maths teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
-<p><a href="/non-uk-teachers/train-to-teach-in-england-as-an-international-student">Non-UK citizens can find out more about training to teach in England</a>.</p>
-<p>If your training provider thinks you need to top up your maths knowledge, you might need to do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training.</p>
-<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to train to teach</a>.</p>
+<p>Find out more about <a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">what qualifications you need to train to teach</a>.</p>
```

Comment:
just to note that on the computing PR, `what` is not included in the link text (which probably works better)?

```suggestion
<p>Find out more about what <a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
```